lib/utils.js

```diff
@@ -0,0 +1,22 @@
+import _ from 'lodash';
+import { exec } from 'teen_process';
+import { fs } from 'appium-support';
+import log from './logger';
+
+
+async function findAppPath (bundleId) {
+  try {
+    const {stdout} = await exec('/usr/bin/mdfind', [
+      `kMDItemCFBundleIdentifier=${bundleId}`
+    ]);
+    const appPath = _.trim(stdout);
```

Comment:
The mdfind can be multiple lines.

```
$ /usr/bin/mdfind kMDItemCFBundleIdentifier=com.appium.AppiumForMac
/Users/kazu/Library/Developer/Xcode/DerivedData/AppiumForMac-dmisgsfgvrtwvoaguoespdimflbc/Build/Products/Debug/AppiumForMac.app
/Applications/AppiumForMac.app
/Users/kazu/GitHub/appium-for-mac/build/Debug/AppiumForMac.app
/Applications/AppiumForMac2.app
```

What about using `split` and get the first one or returning `null` with a message like 'your environment has multiple apps'?

Reply:
Thanks, I didn't know about that. Will take the folder with the most recent access time